client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -127,9 +135,10 @@ class ProductPurchaseFeaturesList extends Component {
 	}
 
 	getPersonalFeatures() {
-		const { selectedSite, planHasDomainCredit } = this.props;
+		const { isPlaceholder, selectedSite, planHasDomainCredit } = this.props;
 
 		return [
+			<HappinessSupportCard isPlaceholder={ isPlaceholder } />,
```

Comment:
We don't seem to take the `isPlaceholder` prop in consideration in `<HappinessSupportCard />`.